streaming_data_types/alarm_al00.py

```diff
@@ -13,9 +13,9 @@
 
 
 class Severity(Enum):
-    OK = 0,
-    MINOR = 1,
-    MAJOR = 2,
+    OK = (0,)
+    MINOR = (1,)
+    MAJOR = (2,)
```

Comment:
Hmm, this looks wrong. I think this was wrong in the last version as well.
Get rid of the commas and dump the parenthesis.